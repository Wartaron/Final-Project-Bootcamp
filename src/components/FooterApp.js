import React from 'react';

export default class FooterApp extends React.Component {
  render() {
    return (
        <footer className="footer">
            <p className="mb-0">Developed by: <strong><a href="https://www.linkedin.com/in/danilo-esteban-plazas-camargo/">Danilo Plazas Camargo</a></strong></p>
            <p className="mb-0">Checkout the GitHub Repository of this project in this <strong><a href="https://github.com/Wartaron/Final-Project-Bootcamp">link</a></strong></p>
        </footer>
    );
  }
}
