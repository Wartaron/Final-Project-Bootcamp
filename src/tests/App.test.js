import React from "react";
// import { Link } from "react-router-dom";
// import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import FooterApp from '../components/FooterApp';

test("renders correctly", () => {
  const tree = renderer
    .create(
      <FooterApp />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
