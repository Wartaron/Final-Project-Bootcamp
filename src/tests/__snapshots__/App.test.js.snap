// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footer"
>
  <p
    className="mb-0"
  >
    Developed by: 
    <strong>
      <a
        href="https://www.linkedin.com/in/danilo-esteban-plazas-camargo/"
      >
        Danilo Plazas Camargo
      </a>
    </strong>
  </p>
  <p
    className="mb-0"
  >
    Checkout the GitHub Repository of this project in this 
    <strong>
      <a
        href="https://github.com/Wartaron/Final-Project-Bootcamp"
      >
        link
      </a>
    </strong>
  </p>
</footer>
`;
